src/manage_sql.c

```diff
@@ -53864,11 +53865,31 @@ manage_set_ldap_info (int enabled, gchar *host, gchar *authdn,
 
   if (cacert)
     {
-      sql ("DELETE FROM meta WHERE name LIKE 'ldap_cacert';");
-      quoted = sql_quote (cacert);
-      sql ("INSERT INTO meta (name, value) VALUES ('ldap_cacert', '%s');",
-           quoted);
-      g_free (quoted);
+      /* check if the certificate is valid */
+      time_t activation_time, expiration_time;
+      gchar *md5_fingerprint, *issuer;
```

Comment:
I think you could leave these out, to save having to free them. `get_certificate_info` only sets them if they're given, and the return code is the same either way.

Reply:
Similar for the times.